docs/tutorials/wiki2/installation.rst

```diff
@@ -242,79 +346,65 @@ On Windows
+For a successful test run, you should see output that ends like this:
+
+.. code-block:: bash
 
    ..
-   ----------------------------------------------------------------------
-   Ran 2 tests in 0.053s
+   2 passed in 0.44 seconds
 
-   OK
 
 Expose test coverage information
 --------------------------------
 
-You can run the ``nosetests`` command to see test coverage information. This
-runs the tests in the same way that ``setup.py test`` does, but provides
-additional "coverage" information, exposing which lines of your project are
-covered by the tests.
+You can run the ``py.test`` command to see test coverage information. This
+runs the tests in the same way that ``py.test`` does, but provides additional
+"coverage" information, exposing which lines of your project are covered by the
+tests.
 
-To get this functionality working, we'll need to install the ``nose`` and
-``coverage`` packages into our ``virtualenv``:
+We've already installed the ``pytest-cov`` package into our ``virtualenv``, so
+we can run the tests with coverage.
 
 On UNIX
 ^^^^^^^
 
 .. code-block:: bash
 
-   $ $VENV/bin/easy_install nose coverage
+   $ $VENV/bin/py.test --cov=tutorial tutorial/tests.py
 
 On Windows
 ^^^^^^^^^^
 
 .. code-block:: ps1con
 
-   c:\pyramidtut\tutorial> %VENV%\Scripts\easy_install nose coverage
-
-Once ``nose`` and ``coverage`` are installed, we can run the tests with
-coverage.
-
-On UNIX
-^^^^^^^
-
-.. code-block:: bash
-
-   $ $VENV/bin/nosetests --cover-package=tutorial --cover-erase --with-coverage
-
-On Windows
-^^^^^^^^^^
-
-.. code-block:: ps1con
-
-   c:\pyramidtut\tutorial> %VENV%\Scripts\nosetests --cover-package=tutorial \
-         --cover-erase --with-coverage
+   c:\pyramidtut\tutorial> %VENV%\Scripts\py.test --cov=tutorial tutorial\tests.py
 
 If successful, you will see output something like this::
 
-   ..
-   Name                         Stmts   Miss  Cover   Missing
-   ----------------------------------------------------------
-   tutorial.py                      8      6    25%   7-12
-   tutorial/models.py              22      0   100%
-   tutorial/models/meta.py          5      0   100%
-   tutorial/models/mymodel.py       8      0   100%
-   tutorial/scripts.py              0      0   100%
-   tutorial/views.py                0      0   100%
-   tutorial/views/default.py       12      0   100%
-   ----------------------------------------------------------
-   TOTAL                           55      6    89%
-   ----------------------------------------------------------------------
-   Ran 2 tests in 0.579s
-
-   OK
+======================== test session starts ========================
+platform Python 3.5.1, pytest-2.9.1, py-1.4.31, pluggy-0.3.1
+rootdir: /Users/stevepiercy/projects/pyramidtut/tutorial, inifile:
+plugins: cov-2.2.1
+collected 2 items
+
+tutorial/tests.py ..
+------------------ coverage: platform Python 3.5.1 ------------------
+Name                               Stmts   Miss  Cover
```

Comment:
Done in https://github.com/Pylons/pyramid/pull/2442/commits/aa51dcc629028fda64ee83340d9b3981be7fbb29